dockerfiles/rbe-ubuntu20-04/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM marketplace.gcr.io/google/ubuntu2004@sha256:9e3ead2510253310bd5b44d254ab810c8f37b19a35716379220fa72b38e8c818
+FROM marketplace.gcr.io/google/ubuntu2004@sha256:a4b4d6c240695a053ab3c437592ec339f608067f307d0e76c9063a6e5c3dc10f
```

Comment:
let's revert this too, since it's one of our RBE images that we would need to migrate https://github.com/buildbuddy-io/buildbuddy-toolchain/blob/dfb7aadff5783bee5f53682144486265ed5e8941/rules.bzl#L129-L131

Reply:
Sure, no prob. Done!